libs/coin-modules/coin-aptos/src/types/index.ts

```diff
@@ -88,6 +88,6 @@ export type AptosMoveResource = {
 };
 
 export type AptosBalance = {
-  asset_type: string;
+  contractAddress: string;
```

Comment:
why change the name?

Reply:
to match the other references in the module code, like when a transaction it's being created on Bridge side...